README.md

```diff
@@ -53,6 +53,9 @@ The client pod in PSM benchmarks includes a
 control plane. The client pod in the proxied case also includes an Envoy
 [sidecar](containers/runtime/sidecar/).
 
+[Prometheus](config/prometheus/README.md) is required to monitor CPU and memory
```

Comment:
I would just say "is used to monitor CPU and memory utilization in PSM benchmarks".